packages/@guardian/source-react-components/src/icons/components/SvgCross.tsx

```diff
@@ -6,31 +6,33 @@ import type { IconProps } from '../types';
 export const SvgCross = ({
 	size,
 	isAnnouncedByScreenReader = false,
-}: IconProps): EmotionJSX.Element => (
-	<>
-		<svg
-			viewBox="0 0 30 30"
-			xmlns="http://www.w3.org/2000/svg"
-			width={size ? iconSize[size] : undefined}
-			aria-hidden={true}
-			focusable={false}
-		>
-			<path
-				fillRule="evenodd"
-				clipRule="evenodd"
-				d="M15.325 17.025l7.225 6.625 1.075-1.075-6.6-7.25 6.6-7.25L22.55 7l-7.225 6.625-7.25-6.6L7 8.1l6.625 7.225L7 22.55l1.075 1.075 7.25-6.6z"
-			/>
-		</svg>
-		{isAnnouncedByScreenReader ? (
-			<span
-				css={css`
-					${visuallyHidden}
-				`}
+}: IconProps): EmotionJSX.Element => {
```

Comment:
The cross appears larger than the height of the letters in the buttons - especially in the default and small sizes - we should probably check with design if they're happy with that: 
https://www.chromatic.com/test?appId=5e933197d94d9c002275a15e&id=62330ead626d2d003a416a3c
![image](https://user-images.githubusercontent.com/15648334/160429392-ae37f67e-7873-4042-9327-b7f86760712a.png)

https://www.chromatic.com/test?appId=5e933197d94d9c002275a15e&id=62330ead626d2d003a416a3d
![image](https://user-images.githubusercontent.com/15648334/160429001-3aea662b-813b-40c4-9086-b9517f9f0754.png)
https://www.chromatic.com/test?appId=5e933197d94d9c002275a15e&id=62330ead626d2d003a416a3e

![image](https://user-images.githubusercontent.com/15648334/160429531-7bf28a6a-e9f7-4486-b26b-f093f5c96c03.png)

Reply:
Good spot. I think this is okay. Akemi deliberately increased the size of this cross (from 16px to 20px) when she migrated the icons into the 24x24 canvas.

This is how the button appears in Figma, in Akemi's comprehensive [Icon Test Page](https://www.figma.com/file/Ai7AELHC6KCz38qKZkvuHo/%E2%97%90-Icons?node-id=1211%3A1096):

<img width="217" alt="Screenshot 2022-03-31 at 11 23 52" src="https://user-images.githubusercontent.com/5931528/161034174-e85cb7d4-edb3-447b-8a30-15bfeb770511.png">